frontend/src/toolbar/flags/FeatureFlags.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { featureFlagsLogic } from '~/toolbar/flags/featureFlagsLogic'
+import { Radio, Space, Switch, Row, Typography } from 'antd'
+import { Collapse } from 'antd'
+
+export function FeatureFlags(): JSX.Element {
+    const { userFlags } = useValues(featureFlagsLogic)
+    const { setOverriddenUserFlag, deleteOverriddenUserFlag } = useActions(featureFlagsLogic)
+
+    return (
+        <div className="toolbar-block">
+            <h1 className="section-title" style={{ paddingTop: 4 }}>
+                Flags ({userFlags.length})
+            </h1>
+            <Collapse>
+                {userFlags.map(({ feature_flag, value_for_user_without_override, override }) => {
+                    const hasVariants = (feature_flag.filters?.multivariate?.variants?.length || 0) > 0
+                    const flagEnabled = override ? !!override?.override_value : !!value_for_user_without_override
+                    const selectedVariant = hasVariants
+                        ? override
+                            ? override?.override_value
+                            : value_for_user_without_override
+                        : undefined
```

Comment:
I think this would be a bit cleaner as a selector in a logic, though also fine to leave here.